analysis/__main__.py

```diff
@@ -106,6 +107,19 @@ def summarize_setting(summary, version, seconds, path, data):
             data = data.split(",")[0].split("(")[0].strip()
             data = data.replace("@TM@", "(TM)").replace("@R@", "(R)").replace("@C@", "(C)")
 
+            if "nvidia" in data.lower() or "geforce" in data.lower() or "quadro" in data.lower():
+                brand = "NVIDIA"
```

Comment:
Nvidia I believe is the correct capitalisation

Reply:
Do you have any source for that?

Both do all cards report NVIDIA, as is everything on https://www.nvidia.com/en-us/ . Can't find a single entry not in full caps.

Reply:
You are correct, I am wrong